site/en/docs/extensions/reference/webRequest/index.md

```diff
@@ -22,6 +20,11 @@ addition. For example:
 }
 ```
 
+If you are building policy-installed extensions for enterprises, and want to use the web request API
+in a blocking fashion, you need to request the `"webRequestBlocking"` permission. As of Chrome 108,
```

Comment:
Would it make sense to move the sentence about that to a separate paragraph? While everything here is technically correct I feel like it could be read as implying that only policy-installed extensions can use `webRequestAuthProvider`, when in reality it's available to all extensions.